ui/main/src/app/modules/feed/feed.component.ts

```
@@ -56,16 +57,20 @@ export class FeedComponent implements OnInit, OnDestroy {
 
     ngOnInit() {
         this.selection$ = SelectedCardStore.getSelectCardIdChanges();
-
+ 
         this.lightCards$ = this.filteredLightCardStore.getFilteredAndSortedLightCards().pipe(
             delay(0), // Solve error: 'Expression has changed after it was checked' --> See https://blog.angular-university.io/angular-debugging/
             map((cards) => {
                 this.totalNumberOfLightsCards = cards.length;
-
+                debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)